src/parser/reader.rs

```diff
@@ -113,36 +115,37 @@ impl<'l> Reader<'l> {
 
     // https://www.w3.org/TR/REC-xml/#NT-NameChar
     pub fn consume_name_char(&mut self) -> bool {
-        self.consume_name_start_char() || self.consume_if(|c| match c {
-            '-'
-            | '.'
-            | '0'...'9'
-            | '\u{B7}'
-            | '\u{0300}'...'\u{036F}'
-            | '\u{203F}'...'\u{2040}' => true,
-            _ => false,
-        })
+        self.consume_name_start_char()
+            || self.consume_if(|c| match c {
```

Comment:
Is it by rustfmt?

Reply:
Yes, commit 858c43eb304084832222ba615c02bffdb4810ffc